src/chain/chain_accumulator.rs

```diff
@@ -293,12 +294,24 @@ mod test {
         ))
     }
 
-    fn random_section_info_sig_payload(rng: &mut MainRng) -> EventSigPayload {
+    fn random_section_info_sig_payload(
+        rng: &mut MainRng,
+    ) -> (EventSigPayload, RealBlsEventSigPayload, RealBlsPublicKeySet) {
+        let bls_keys = generate_bls_threshold_secret_key(rng, 2);
```

Comment:
why choosing 2 ?
and better make it as a const ?

Reply:
pretty arbitrary, allow to have 2 different keys. Will make it a named local